rfcs/077-enriching-addressables-with-works/README.md

```diff
@@ -0,0 +1,161 @@
+
+- `id`
+- `title`
+- `type`
+- `thumbnail`
+- `production`
+- `contributors`
+- `workType`
+- `availabilities`
+
+These are sufficient to render the desired Works previews.
+
+### New endpoint
+
+The new endpoint will be `https://api.wellcomecollection.org/content/v0/all/{id}`
+
+Its response will return:
+
+```json
+{
+  "type": "Article",
+  "id": "Z-FyfxEAACIAwNl_",
+  "uid": "waking-the-dead",
+  "title": "Waking the dead",
+  "works": [
+    {
+      "id": "a2239muq",
+      "title": "Ueber den Krebs der Nasenhöhle ... / vorgelegt von Hermann Wolter.",
+      "type": "Work",
+      "thumbnail": {
+        "url": "https://iiif.wellcomecollection.org/thumbs/b30598977_0001.jp2/full/!200,200/0/default.jpg",
+        "license": {
+          "id": "inc",
+          "label": "In copyright",
+          "url": "http://rightsstatements.org/vocab/InC/1.0/",
+          "type": "License"
+        }
+      },
+      "production": [
+        {
+          "label": "Leipzig : Bruno Georgi, 1900.",
+          "agents": [
+            {
+              "label": "Bruno Georgi",
+              "type": "Agent"
+            }
+          ],
+          "dates": [
+            {
+              "label": "1900",
+              "type": "Period"
+            }
+          ],
+          "type": "ProductionEvent",
+          "places": [
+            {
+              "label": "Leipzig",
+              "type": "Place"
+            }
+          ]
+        }
+      ],
+      "contributors": [
+        {
+          "agent": {
+            "id": "eq9qvtwy",
+            "identifiers": [
+              {
+                "identifierType": {
+                  "id": "label-derived",
+                  "label": "Identifier derived from the label of the referent",
+                  "type": "IdentifierType"
+                },
+                "value": "wolter, hermann (wilhelm victor hermann), 1868-",
+                "type": "Identifier"
+              }
+            ],
+            "label": "Wolter, Hermann (Wilhelm Victor Hermann), 1868-",
+            "type": "Person"
+          },
+          "roles": [],
+          "primary": true,
+          "type": "Contributor"
+        },
+        {
+          "agent": {
+            "id": "un9788h3",
+            "identifiers": [
+              {
+                "identifierType": {
+                  "id": "lc-names",
+                  "label": "Library of Congress Name authority records",
+                  "type": "IdentifierType"
+                },
+                "value": "n85086716",
+                "type": "Identifier"
+              }
+            ],
+            "label": "Universität Leipzig (1409-1953)",
+            "type": "Organisation"
+          },
+          "roles": [],
+          "primary": false,
+          "type": "Contributor"
+        }
+      ]
+    }
+  ]
+}
+```
```

Comment:
I think we could trim a lot of it off to really just have what we need for the UI

Reply:
Yeah that was one of the questions I had; whether we should keep this the same as what we get from the Catalogue API or not. But I think you're right and we should do all the transforming up front and only send what we need. I'll amend it.

Reply:
Ah I see! That's what we've gone with with all the other endpoints, so yes 👍